packages/react-instantsearch/src/core/createConnector.js

```diff
@@ -143,9 +144,10 @@ export default function createConnector(connectorDesc) {
         error,
         widgets,
         metadata,
+        resultsFacetValues,
       } = store.getState();
       const searchState = {results, searching, error};
-      return connectorDesc.getProvidedProps.call(this, props, widgets, searchState, metadata);
+      return connectorDesc.getProvidedProps.call(this, props, widgets, searchState, metadata, resultsFacetValues);
```

Comment:
@vvo I'm not sure to get the question :D you propose to merge the results of the search for facet values (`resultsFacetValues`) into the results object (`results`)?